orc8r/tools/helm/package_lf.sh

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# package.sh packages and publishes orc8r helm charts to a private git repo
+
+set -e -o pipefail
+
+# Valid deployment types
+FWA="fwa"
+FFWA="federated_fwa"
+ALL="all"
+ORC8R_VERSION="1.4"
```

Comment:
@ShubhamTatvamasi : I know that there was some discussion about the versions and versioning of the helm charts. Is that Version right here?

Reply:
Now it should be 1.8

Reply:
done.